### PR TITLE
Task5 update delete

### PR DESCRIPTION
**Pull request by ArBond**

### CODE REVIEW

File: lab-SQL/task5_UPDATE-DELETE.sql [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+		person.ModifiedDate,
+		NULL
+FROM dbo.Person person
+JOIN HumanResources.Employee employee
+ON person.BusinessEntityId = employee.BusinessEntityID
+GO
+
+--point 4.
+UPDATE dbo.Person_New
+SET	Salutation = CONCAT_WS(' ', Title, FirstName);
```
Comment (Glareone) [inline lab-SQL/task5_UPDATE-DELETE.sql]: tab instead of space

File: lab-SQL/task5_UPDATE-DELETE.sql [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+GO
+
+--point 5.
```
Comment (Glareone) [inline lab-SQL/task5_UPDATE-DELETE.sql]: also could you please add comments like you did before.

File: lab-SQL/task5_UPDATE-DELETE.sql (hunk 1 above)
Comment (ArBond) [reply lab-SQL/task5_UPDATE-DELETE.sql]: done

File: lab-SQL/task5_UPDATE-DELETE.sql (hunk 2 above)
Comment (ArBond) [reply lab-SQL/task5_UPDATE-DELETE.sql]: agreed to partially miss ;)
done

**Pull request merged**